site/docs/reference/Connectors/capture-connectors/amazon-s3.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 2
+---
+# Amazon S3
+
+This connector captures data from an Amazon S3 bucket.
+
+[`ghcr.io/estuary/source-s3:dev`](https://ghcr.io/estuary/source-s3:dev) provides the latest connector image when using the Flow GitOps environment. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, either your S3 bucket must be public,
+or you must have access via a root or [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html).
+
+* For public buckets, verify that the [access policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-overview.html#access-control-resources-manage-permissions-basics) allows anonymous reads.
+* For buckets accessed by a user account, you'll need the AWS **access key** and **secret access key** for the user.
+See the [AWS blog](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) for help finding these credentials.
+
+## Configuration
+
+There are various ways to configure and implement connectors. See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about these methods. The values and code sample in this section provide configuration details specific to the Amazon S3 source connector.
+
+:::tip
+You might organize your S3 bucket using [prefixes](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html) to emulate a directory structure.
+This connector can use prefixes in various ways, giving you precise control over how datasets are captured into Flow.
+You can specify a prefix in the endpoint configuration to limit the overall scope of the capture within your bucket.
+You'll also specify prefixes on a per-binding basis, allowing you to map each prefix to a distinct Flow collection.
+
+Alternatively, you can capture the entire bucket by omitting `prefix` in the endpoint configuration and
+setting `stream` to the name of the bucket.
+:::
+
+### Values
+
+#### Endpoint
+
+| Value | Name| Description | Type | Required/Default |
```

Comment:
FYI, and I'm not sure how to best document this, but: `bucket` and `prefix` are used / required for discovery but aren't actually used when the connector is running. Instead, the connector takes each binding `stream` and destructures the bucket and prefix from the `stream` name.

It's even possible to have a single capture which binds datasets living across _multiple_ buckets,
but it's on the user to create correct `stream` names for each binding.

This is true of S3 & GCS, and will eventually be true of Azure Blob Store when we inevitably build that connector.